packages/opentelemetry-exporter/README.md

```diff
@@ -0,0 +1,67 @@
+# @instana/opentelemetry-exporter
```

Comment:
https://github.com/open-telemetry/opentelemetry-js#supported-runtimes

Maybe something to mention in our docs too? We support 6 & 8, Otel doesn't officially.

Reply:
I am not entirely sure to be honest.
Essentially, this package is a span exporter that actually depends on the OTel SDK, so being compatible to their supported runtimes is something that their users inevitably have to respect anyway.
The only dependency from our stack is actually the communication to the backend, through the `@instana/serverless` package.
I see no harm at mentioning the version compatibility, but as far as I can see, it doesn't mean much.

Reply:
My thinking was that our customers start to use Otel and might use an older version (as a hint for our customers).
But maybe that is not a use case for our customers. Also not sure :)

Reply:
I am adding a small section with links to both our docs and OTel docs where supported versions are described.
I also add a note concerning Instana customers who wish to switch to Otel.